data/scripts/cloud-remediation.sh

```diff
@@ -0,0 +1,6 @@
+# Run the image hardening script and then remove the package after completion
+/usr/bin/bash -e /opt/pct/cloud-remediation.sh
+if [[ $? -ne 0 ]] ; then
+    exit 1
+fi
```

Comment:
Checking the exit status is not necessary. `config.sh` sets the abort-on-error flag. I'm not sure about running `cloud-remediation.sh` itself with `-e`. This could have side-effects. I'd be reluctant to force it on external scripts.

Reply:
I will adjust per your comments push it up shortly

Reply:
In testing I think we need the -e. The remediation script which is generated from here 

https://github.com/brett060102/content/tree/pcs-hardening/products/sle15/profiles

does not use exit codes in case a remediation is not applied as this could mean there was nothing to do.  I tested this with and without the -e and in only aborts correctly when the -e is present

Reply:
A had a look at the script that's in your IBS project and it uses `false` statements to indicate error situations without forcing abort, so it seems it's actually designed to be run with `-e`. I just mentioned possible side-effects because if a script isn't designed for that, it may break, but in this case I think we do want to run it with `-e`.

As a side note, wouldn't it be better to generate the script in the build instead of simply packaging the script file?

Reply:
Joe, the idea is the cloud-remediation is only needed until this https://github.com/brett060102/content/tree/pcs-hardening get merged upstream.  We have no ETA at to that time frame. Once that is done however, the profiles listed here https://github.com/ComplianceAsCode/content are able to be remediated using the oscap tools.  When that occurs, we simply update the scriptlet in keg recipes. I do not think it would be optimal to have the script generated as part of he cloud-remediation  package build, but that conversation can take place when iI do the 'sr' for cloud-remediation.  Any reason not to merge this change here now.?

Reply:
Ah ok, I didn't realize this was just a temporary thing. In that case we might as well keep it in `keg-recipes` for the time being. If that package just contains the generated script, there isn't really a benefit IMO. Anyhow, PR is fine for merge from my POV.